Components/Home/Home.css

```diff
@@ -27,8 +33,7 @@
 		margin: 0;
 	}
 	.content{
-		padding-left: 32px;
```

Comment:
I don't see where negative margin is used? I replaced the manual padding with LayoutGrid, which has by default 24px padding on desktop and 16px padding on mobile. I can change it to 32px on desktop if you'd like that.